testdata/config.lua

```diff
@@ -0,0 +1,40 @@
+-- Do not set listen for now so connector won't be
+-- able to send requests until everything is configured.
+local auth_type = os.getenv("TEST_TNT_AUTH_TYPE")
+if auth_type == "auto" then
+    auth_type = nil
+end
+
+box.cfg{
+    auth_type = auth_type,
+    work_dir = os.getenv("TEST_TNT_WORK_DIR"),
+    memtx_use_mvcc_engine = os.getenv("TEST_TNT_MEMTX_USE_MVCC_ENGINE") == 'true' or nil,
```

Comment:
I think we don't `memtx_use_mvcc_engine` here (it is required by streams tests).